Apps/phgreynoise/greynoise_connector.py

```diff
@@ -106,8 +106,9 @@ def get_session(self):
         return self._session
 
     def _make_rest_call(
-        self, action_result, method, *args, error_on_404=False, **kwargs
+        self, action_result, method, *args, **kwargs
     ):
+        error_on_404 = False
```

Comment:
Hii @hl-tstrawbridge,

We found one thing is that in the "_make_rest_call" method the argument "error_on_404" is placed after the *args arguments which are caused the pipeline error it gives the syntax error. And we also noticed that the "error_on_404" argument is not passing by any of the calls of make_rest_call. So we moved this "error_on_404" argument inside the method as a variable. So could you please confirm from your side?

Reply:
We've used the free account as mentioned in this PR(https://github.com/phantomcyber/phantom-apps/pull/654) to test the basic positive TCs.

Hence, approving the PR